EventEnabledInsurance/ProjectionClaims/src/main/java/com/ibm/cp4i/demos/eei/projectionclaims/MyHttpHandler.java

```diff
@@ -52,7 +51,7 @@ public static void createTableForAllData(JsonNode table, StringBuilder contentBu
             contentBuilder.append
             (
                 "<tr>" +
-                    "<td><a href=\"quoteid=" + quoteid + "\">" + quoteid + "</a></th>" +
+                    "<td><a href=\"/?quoteid=" + quoteid + "\">" + quoteid + "</a></th>" +
```

Comment:
It looks like this bit of the task hasn't been done:
<img width="628" alt="image" src="https://user-images.githubusercontent.com/64848511/97285719-754bbe00-183a-11eb-8fbd-6954ea834958.png">

I guess that means the GET and the output in the Projection Claims app will return different names for the fields than is used by the POST. I.e. `licenseplate` rather than `licensePlate`?

Reply:
the GET output now follows the POST